content/requirements.md

```diff
@@ -14,4 +17,4 @@ Git for Windows requires Windows 11 on the ARM64 CPU architecture.
 
 Why?
 
-The POSIX emulation layer mentioned above does not yet exist for ARM64, so we need x86_64 emulation to run the x86_64 version. The required x86_64 emulation is not available prior to Windows 11. While an i686 version exists, that is [on a deprection timeline](https://gitforwindows.org/32-bit.html).
```

Comment:
Whitespace change only?

Reply:
Yes. The file had no trailing newline before the move (and the edit). I guess I'll just keep it this way, as `vim` insists on adding a newline there and I am prone to forget to remove it.